### PR TITLE
Update with green water variants

### PR DESCRIPTION
**Pull request by Rockstar04**

### CODE REVIEW

**Comment by Rockstar04** [comment]
Saw this as a request in the Landfill thread on the forums

File: src/control.lua [hunk 1]
```diff
@@ -7,7 +7,9 @@ local bombs
 local replaceableTiles =
 {
   ["water"] = "grass",
-  ["deepwater"] = "grass"
+  ["deepwater"] = "grass".
```
Comment (Rseding91) [inline src/control.lua]: I think that "." is meant to be a ",".

Comment (Rockstar04) [inline src/control.lua]: Yes, both of those lines are botched.

**Comment by Rockstar04** [comment]
I blame the github editor (no, it was my fault).

Updated the PR

**Comment by Rseding91** [comment]
So, I forgot to look back at this after I got home from work Monday :) I'll try to get back here tonight when I'm home and available to do a version increment with these changes.

**Pull request merged**